src/game/server/gamecontext.cpp

```diff
@@ -712,17 +712,35 @@ void CGameContext::OnClientConnected(int ClientID)
 
 bool CGameContext::OnClientDrop(int ClientID, const char *pReason, bool Force)
 {
+	CPlayer *pPlayer = m_apPlayers[ClientID];
 	m_pController->t_stats->on_drop(ClientID, pReason);
-	
-	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force) return false;
 
+	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force)
```

Comment:
You could use pPlayer here too, if you got it anyway...

Reply:
oggi